### DISCUSSION TITLE
Buy a coffee link

### DISCUSSION THREAD

**Discussion by austincollinpena** [arthelokyo/astrowind#3] [Ideas]
If you add a "buy a coffee" link I'll happily pay some. You've saved me a ton of time

**Comment by prototypa** [comment]
Hi @austincollinpena

I'm glad AstroWind was helpful to you. I'll be thinking about the "buy a coffee" link, it may be a good idea to add more fuel to this.

The template still requires some improvements, if in the process of making your project you detect new features that the template could have, do not hesitate to write, that would be very helpful. Thanks

**Comment by prototypa** [comment]
Hi @austincollinpena 😉
https://www.buymeacoffee.com/onwidget
